### PR TITLE
jabber:iq:gateway and XEP-0106 escaping

### PR DESCRIPTION
**Pull request by singpolyma**
Related to https://github.com/movim/movim/issues/367

### CODE REVIEW

**Comment by singpolyma** [comment]
@edhelas this one is ready for review now :)

File: src/Moxl/Stanza/IqGateway.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Moxl\Stanza;
+
+use Moxl\Utils;
+
+class IqGateway {
```
Comment (edhelas) [inline src/Moxl/Stanza/IqGateway.php]: Don't forget to put the { inline here

File: src/Moxl/Xec/Action/IqGateway/Set.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    public function setExtra($extra)
+    {
+        $this->_extra = $extra;
+        return $this;
+    }
+
+    public function handle($stanza, $parent = false)
+    {
+        $this->prepare($stanza, $parent);
+        $this->pack((object)[
```
Comment (edhelas) [inline src/Moxl/Xec/Action/IqGateway/Set.php]: No need to cast to object here you can directly past the array

**Comment by singpolyma** [comment]
@edhelas I believe I have made all your suggested changes :)

**Pull request merged**